### PR TITLE
The checkpoint (InstantOn) related message IDs need fixed.

### PR DESCRIPTION
**Pull request by tjwatson**

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
@OpenLiberty/message-reviewer All these messages had already been approved from previous PRs.  But for various reasons they either ended up getting their message ID used by other new messages or even completely removed (accidentally) in other cases.  This PR fixes up the duplicate use of message IDs as well as add back the messages that got removed.

**Review by sebratton — DISMISSED** [review]

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -736,25 +736,29 @@ error.cannotset.securitymanager.jdk18=CWWKE0956E: Failed to set the Java Securit
 error.cannotset.securitymanager.jdk18.explanation=Java Security Manager could not be enabled because the system is using Java 18, which by default disallows use of a Java Security Manager.
 error.cannotset.securitymanager.jdk18.useraction=Start the server with the JVM argument -Djava.security.manager=allow.
 
-error.checkpoint.delete=CWWKE0955E: The server checkpoint request failed because the existing checkpoint image cannot be deleted from the {0} directory.
+error.checkpoint.workarea.restore=CWWKE0957E: The checkpoint server workarea directory cannot be restored from the {0} directory.
+error.checkpoint.workarea.restore.explanation=An error occurred restoring the checkpoint workarea directory.
+error.checkpoint.workarea.restore.useraction=Ensure the user has sufficient permissions to restore the checkpoint workarea. Reissue the command with a user that has sufficient permissions.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]:
```suggestion
error.checkpoint.workarea.restore.useraction=Ensure that the user has sufficient permissions to restore the checkpoint workarea. Reissue the command with a user that has sufficient permissions.
```

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 2]
```diff
@@ -736,25 +736,29 @@ error.cannotset.securitymanager.jdk18=CWWKE0956E: Failed to set the Java Securit
...
+error.checkpoint.securitymanager.not.supported.useraction=Remove the websphere.java.security property from the bootstrap.properties file.
+
+error.checkpoint.delete=CWWKE0959E: The server checkpoint request failed because the existing checkpoint image cannot be deleted from the {0} directory.
 error.checkpoint.delete.explanation=A checkpoint request for the server failed because an existing checkpoint image cannot be deleted.
 error.checkpoint.delete.useraction=Ensure the user has permission to delete the existing directory. Reissue the command with a user that has sufficient permissions.
 
-error.checkpoint.workarea.backup=CWWKE0956E: The server checkpoint request failed because the {0} directory cannot be backed up.
+error.checkpoint.workarea.backup=CWWKE0960E: The server checkpoint request failed because the {0} directory cannot be backed up.
 error.checkpoint.workarea.backup.explanation=An error occurred backing up the checkpoint server workarea directory.
 error.checkpoint.workarea.backup.useraction=Ensure the user has sufficient permissions to back up the checkpoint workarea directory. Reissue the command with a user that has sufficient permissions.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]:
```suggestion
error.checkpoint.workarea.backup.useraction=Ensure that the user has sufficient permissions to back up the checkpoint workarea directory. Reissue the command with a user that has sufficient permissions.
```

**Comment by tjwatson** [comment]
@ramkumar-k-9286 I applied your suggestions.

**Comment by tjwatson** [comment]
#libby

**Review by jaridk — APPROVED** [review]

**Pull request merged**